routes/accounts.js

```diff
@@ -80,10 +81,22 @@ function accountRoutes (server, options, next) {
     handler: function (request, reply) {
       var sessionId = toBearerToken(request)
 
-      return accounts.findAll({
-        db: options.db,
-        bearerToken: sessionId,
-        include: request.query.include
+      admins.validateSession(sessionId)
+
+      .catch(function (error) {
+        if (error.toString() === dbErrors.MISSING_DOC.toString()) {
```

Comment:
I think we can compare the `.name` property only: `error.name === dbErrors.name`